sycl/source/detail/stream_impl.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include <CL/sycl/detail/stream_impl.hpp>
 #include <cstdio>
+#include <detail/scheduler/scheduler.hpp>
```

Comment:
According to the coding style guidelines system header includes should go last. https://llvm.org/docs/CodingStandards.html#include-style
```c++
#include <CL/sycl/detail/stream_impl.hpp>
#include <detail/scheduler/scheduler.hpp>

#include <cstdio>
```